src/Proxy.hs

```diff
@@ -711,6 +740,28 @@ getSimpleTransactionStatus i trHash = do
         (h:r)
           | all (h==) r -> return h
           | otherwise -> return ["outcome" .= String "ambiguous"]
+    updateEventToMaybeValue :: Event -> Maybe Value
```

Comment:
It would be useful to document what this does, and that it should only be applied to specific events coming from smart contract updates.

Reply:
Done.